src/parameter_bridge.cpp

```diff
@@ -85,9 +95,93 @@ int main(int argc, char * argv[])
     }
   } else {
     fprintf(
-      stderr, "The parameter '%s' either doesn't exist or isn't an array\n", parameter_name);
+      stderr, "The parameter '%s' either doesn't exist or isn't an array\n", topics_parameter_name);
   }
 
+  // ROS 1 Services in ROS 2
+  XmlRpc::XmlRpcValue services_1_to_2;
+  if (
+    ros1_node.getParam(services_1_to_2_parameter_name, services_1_to_2) &&
+    services_1_to_2.getType() == XmlRpc::XmlRpcValue::TypeArray)
+  {
+    for (size_t i = 0; i < static_cast<size_t>(services_1_to_2.size()); ++i) {
+      std::string service_name = static_cast<std::string>(services_1_to_2[i]["service"]);
+      std::string package_name = static_cast<std::string>(services_1_to_2[i]["package"]);
+      std::string type_name = static_cast<std::string>(services_1_to_2[i]["type"]);
+      printf(
+        "Trying to create bridge for ROS1 service '%s' "
+        "with package '%s' and type '%s'\n",
+        service_name.c_str(), package_name.c_str(), type_name.c_str());
+
+      auto factory = ros1_bridge::get_service_factory(
+        "ros2", package_name, type_name);
+      if (factory) {
+        try {
+          service_bridges_1_to_2.push_back(factory->service_bridge_1_to_2(ros1_node, ros2_node, service_name));
+          printf("Created 1 to 2 bridge for service %s\n", service_name.c_str());
+        } catch (std::runtime_error & e) {
+          fprintf(
+            stderr,
+            "failed to create bridge ROS1 service '%s' "
```

Comment:
In the printed messages please use `ROS 1` / `ROS 2` (with a space between `ROS` and the number) as in the existing code.